libraries/core-react/src/components/Chip/Chip.tokens.ts

```diff
@@ -29,66 +30,93 @@ const {
   },
 } = tokens
 
-export const chip = {
-  enabled: {
-    background: backgroundColor,
-    height: '24px',
-    border: {
-      radius: borderRadius,
-    },
-    spacings: {
-      left: x_small,
-      right: small,
-    },
-    typography: {
-      ...chipTypography,
-      color: primaryColor,
-    },
-    icon: {
-      height: medium,
-      width: medium,
-      border: {
-        radius: borderRadius,
+export type ChipToken = ComponentToken & {
```

Comment:
Checkout  `CircularProgress` [tokens](https://github.com/equinor/design-system/blob/c6433df9f32fbb2c88bca0e768eda58940551baf/libraries/core-react/src/components/Progress/Circular/CircularProgress.tokens.ts#L20) on suggestion on how to make this easier